src/utils/http-api.js

```diff
@@ -41,17 +41,21 @@ export function getInputs (url, bundle, inputs = {}) {
     .then(parsedBody => JSDP.deserialize(parsedBody));
 }
 
-export function runJob(url, bundle, inputs = {}) {
+export function runJob(url, bundle, inputs = {}, addDebugLogs = false) {
+    
+    var body = {
+        bundle,
+        inputs: JSDP.serialize(inputs, { toObject: true }),
+        debug: addDebugLogs
+    };
+    
```

Comment:
I would say either keep body the makeRequest statement or do the same thing in `getInputs` (for consistency sake).